src/endpoint/sts/ops/sts_post_assume_role.js

```diff
@@ -19,9 +21,9 @@ async function assume_role(req) {
         throw new StsError(StsError.InternalFailure);
     }
     // Temporary credentials are NOT stored in noobaa
-    // TODO: need to generate session token and store in it 
-    // the temporary credentials and expiry
+    // The generated session token will store in it the temporary credentials and expiry and the role's access key
     const access_keys = await req.sts_sdk.generate_temp_access_keys();
+    const expiry = process.env.STS_SESSION_TOKEN_EXPIRY || config.STS_DEFAULT_SESSION_TOKEN_EXPIRY_MS;
```

Comment:
I would move this "OR" to the ```config.js```
It is better to have a single point of edits for the values instead of having "OR" in multiple points of code if we would want to know expiry in different parts of the code
Looking at a single source of truth seems like a simpler solution to maintain

Reply:
I eventually deleted this env var since I could just change it in the config